shared/js/background/chrome-events.es6.js

```diff
@@ -237,6 +243,12 @@ chrome.webNavigation.onCommitted.addListener(details => {
 // Replace UserAgent header on third party requests.
 chrome.webRequest.onBeforeSendHeaders.addListener(
     function spoofUserAgentHeader (e) {
+        let tab = tabManager.get({ tabId: e.tabId })
+        if (tab.site.isBroken) {
```

Comment:
Nice, good call here.

Reply:
Actually, mind checking that tab is defined here? And in the one above.
`if (tab && tab.site.isBroken) {`
Sorry, I just realized this was throwing some errors on the initial extension load.